public/docs/_examples/user-input/dart/lib/app_component.dart

```diff
@@ -1,16 +1,14 @@
 // #docregion
-library user_input.app_component;
-
 import 'package:angular2/angular2.dart';
 import 'package:user_input/click_me_component.dart';
```

Comment:
Maybe you should use relative imports to keep consistency with the other dart examples.